### PR TITLE
Brazilian Readme added.

### PR DESCRIPTION
**Pull request by lnrego**
Brazilian Readme added, formatted

### CODE REVIEW

**Comment by lnrego** [comment]
Hi @loafyloaf, any Idea why it's getting an error?

**Comment by sandmman** [comment]
@inrego This is a security feature. PRs opened by users without write access will fail the travis build immediately in order to prevent them from using store testing credentials maliciously.

**Comment by lnrego** [comment]
@sandmman 

How am I supposed to upload this file then?

**Comment by Inrego** [comment]
Just FYI, my username is with uppercase i. The other guy in this PR is with lowercase L.

File: README-pt.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+```
+
+## Configurando o aplicativo Swift-Enterprise-Demo
+
+
+O arquivo de configuração `cloud_config.json` localizado na pasta raiz do repositório do aplicativo deve ser atualizado antes de você começar a usar o aplicativo.
+
+```bash
+$ cat cloud_config.json
+{ "name": "SwiftEnterpriseDemo",
```
Comment (sandmman) [inline README-pt.md]: This configuration format is out of date.

**Pull request closed without merging**